bld/namelist_files/namelist_defaults_overall.xml

```diff
@@ -75,6 +75,7 @@ determine default values for namelists.
 <mask hgrid="ne240"               >gx1v6</mask>
 <mask hgrid="ne30"                >gx1v6</mask>
 
+<mask hgrid="0.125nldas2"         >nldas2</mask>
```

Comment:
I couldn't tell if this was actually needed, but it seems to be done for other grids, so I added it.

Reply:
It is right now. It listens to this, but needs to be changed to listen to the actual grid.